src/SignupRouter.web.js

```diff
@@ -18,7 +18,7 @@ import { getOriginalScreenHeight } from './lib/utils/Orientation'
 
 const initialRouteName = 'Auth' // isMobileSafari && isWebApp ? 'IOSWebAppSignIn' : 'Auth'
 
-const AuthType = isWeb && Config.torusEnabled ? AuthTorus : Auth
+const AuthType = Config.torusEnabled ? AuthTorus : Auth
```

Comment:
Can't we just keep the same file with this same condition but instead of using the imports at the top use require() directly in the const?
Like:
`const AuthType = isWeb && Config.torusEnabled ? require('path/to/authTorus') : require('path/to/auth')`

What do you think @sirpy ?

Reply:
I tried with this and seems to work well, what do you think guys?
`const AuthTorus = lazy(() => isWeb && import('./components/auth/torus/AuthTorus'))`
`const AuthType = isWeb && Config.torusEnabled ? AuthTorus : Auth`